Plugins/SpatialGDK/SpatialGDKEditorToolbar/Source/SpatialGDKEditorToolbar/Private/InteropCodeGenerator/SpatialGDKEditorInteropCodeGenerator.cpp

```diff
@@ -188,63 +211,30 @@ void GenerateInteropFromClasses(const ClassHeaderMap& Classes, const FString& Co
 		const TArray<FString>& TypeBindingHeaders = ClassHeaderList.Value;
 		bool bIsSingleton = SingletonList.Find(ClassHeaderList.Key) != INDEX_NONE;
 
-		ComponentId += GenerateCompleteSchemaFromClass(CombinedSchemaPath, CombinedForwardingCodePath, ComponentId, Class, TypeBindingHeaders, bIsSingleton);
+		ComponentId += GenerateCompleteSchemaFromClass(CombinedSchemaPath, CombinedForwardingCodePath, ComponentId, Class, TypeBindingHeaders, bIsSingleton, Classes);
 	}
 }
 
-bool RunProcess(const FString& Command, const FString& Arguments)
+bool RunProcess(const FString ExecutablePath, FString Arguments)
```

Comment:
`FMonitoredProcess` can take in const ref strings. Could we do that here?

Reply:
it should, oversight on my end while using the refactoring tool in VA.